src/TritonRoute/src/dr/FlexGridGraph_maze.cpp

```diff
@@ -552,14 +552,14 @@ frCoord FlexGridGraph::getCostsNDR(frMIdx gridX,
   frMIdx startX, startY, endX, endY;
   frCoord r, x1, x2, y1, y2;
   frCoord sp, wext;
-  frCoord layerWidth = max((int) layer->getMinWidth(), ndr_->getWidth(gridZ));
+  frCoord layerWidth = max((int) layer->getWidth(), ndr_->getWidth(gridZ));
   sp = max(ndr_->getSpacing(gridZ),
-           getMinSpacingValue(layer, layerWidth, layer->getMinWidth(), 0));
-  wext = max(ndr_->getWireExtension(gridZ), (int) layer->getMinWidth() / 2)
-         - layer->getMinWidth() / 2;
+           getMinSpacingValue(layer, layerWidth, layer->getWidth(), 0));
+  wext = max(ndr_->getWireExtension(gridZ), (int) layer->getWidth() / 2)
+         - layer->getWidth() / 2;
```

Comment:
For my own understanding how does changing from minWidth to Width effect the router. It seems like a big change so I'm wondering if the original implementation was a bug, or if this implementation simply gives better results.

Reply:
Often they are equal.  Width is the preferred width to use and is more correct.

Reply:
I own the logic of this piece of code and I noticed it was wrong.